### PR TITLE
Task/initial implementation

### PR DESCRIPTION
**Pull request by DeanTang**
Created grid and blocks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # FifteenLua
 A game of Fifteen using Lua
```
Comment (PraxTheory) [inline README.md]: Add a dot here please

**Review by PraxTheory — COMMENTED** [review]
Can we please get this changed? It hasn't been done.

File: README.md (hunk 1 above)
Comment (DeanTang) [reply README.md]: done

File: README.md
Comment (PraxTheory) [inline README.md:1]: Please add another dot after Lawrence. 

Regards

**Review by PraxTheory — COMMENTED** [review]
pelase

File: README.md
Comment (DeanTang) [reply README.md:1]: added

**Review by PraxTheory — COMMENTED** [review]
.

**Pull request merged**